experimental-features.js

```diff
@@ -0,0 +1,10 @@
+module.exports = [
+  {
+    name: 'ComponentName',
```

Comment:
what does the value for `name` represent?

Reply:
It's the component name - like Badge or Alert

Reply:
Oh sorry, I mean what is it's function in indicating a feature as experimental? I expected the value to be used as the link name in the navigation, but looks like that link is generated from the component/layout/utility name.

Reply:
Good catch - I am not directly using this value anywhere, but I left it in since I think it more clearly articulates in the json values what is being indicated.  I can remove it - but it seemed to improve readability of the file when it was in there.

Reply:
works for me either way.